lib.nix

```diff
@@ -1,24 +1,32 @@
 let
+  todo = builtins.fromJSON (builtins.readFile ./nixpkgs-src.json);
+  fixme = { account, project, rev, sha256, sha256unpacked }:
+    pkgs.fetchFromGitHub {
+      owner = account;
+      repo = project;
```

Comment:
I guess, we could just call them `owner`, and `repo` in the json file, and be done with `inherit` them, or even just calling the function with the map.